profiles/logical/current-version.nix

```
@@ -0,0 +1,7 @@
+{ ... }: {
+  environment.etc.holo-nixpkgs-version =
+    let path = (<holo-nixpkgs> + "/.git-version"); in
+    if builtins.pathExists path
+    then { source = path; }
+    else { text = "--failiure--"; };
```

```suggestion
{ lib, ... }: {
  environment.etc.holo-nixpkgs-version =
    let path = (<holo-nixpkgs> + "/.git-version"); in
    lib.optionalAttrs (builtins.pathExists path) { source = path; };
```

If `/etc/holo-nixpkgs-version` exists, it should contain a valid version -- otherwise, it should not exist, IMHO.